### ISSUE TITLE
Yearbook Sample Confirmation: Search for child OCR from parent UV should return results

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#44]
## SUMMARY

During the latest client iteration meeting, Randy demoed a yearbook sample. 

The user is able to search for the child's OCR and find a highlighted match, when searching within the child's UV search. 

However, when searching for that same term in the parent's UV search, no matches are found. 

Their observations is from their sever.

This appears to be working with [other sample data](https://lv-hyku-staging.notch8.cloud/concern/texts/d67e1140-d694-40aa-ba5c-c87ce00118e9?locale=en), so I've asked the client to import this example onto our staging. 

Link to referenced work on our server: https://lv-hyku-staging.notch8.cloud/concern/texts/b1cfd03a-908d-479b-be5d-fbea6ce44b29?locale=en

## ACCEPTANCE CRITERIA

- [x] The user should be able to search for child OCR and metadata from the parent UV, and return matches if found.

## SCREENSHOTS

1. Search for “Revember” parent vs child: misspelling is intentional because that’s what’s listed in the available ocr txt

child: https://lv-hyku-staging.notch8.cloud/concern/parent/41c06817-5601-4012-af22-47b08728552b/images/822233f3-18c6-4a27-a5d6-e03423caadab

available ocr: https://lv-hyku-staging.notch8.cloud/downloads/18faf537-a00b-4bfb-bba2-108e21ae5148?locale=en&file=txt

![Image](https://user-images.githubusercontent.com/1054448/203152561-039946e9-e270-43e7-980b-b61cdc86afc1.png)

parent: https://lv-hyku-staging.notch8.cloud/concern/images/41c06817-5601-4012-af22-47b08728552b?locale=en

![Image](https://user-images.githubusercontent.com/1054448/203152629-9e3ee03a-8f24-4ad7-b101-f7604c857ea0.png)

2. search for 'president'

CHILD: https://lv-hyku-staging.notch8.cloud/concern/parent/b1cfd03a-908d-479b-be5d-fbea6ce44b29/texts/a3cac71e-0676-453c-b0b1-6df2f8f6d9d7

![Image](https://user-images.githubusercontent.com/1054448/203152813-5fb7d8b0-bd3b-4e2c-8ff5-34ecc5c3726e.png)

PARENT: https://lv-hyku-staging.notch8.cloud/concern/texts/b1cfd03a-908d-479b-be5d-fbea6ce44b29?locale=en

![Image](https://user-images.githubusercontent.com/1054448/203152877-82e7c312-259f-475c-bed7-811c98221415.png)

3. search for "engraved" 

**Sample Data**
: https://drive.google.com/file/d/1y-m6QhE_s6r_6LMVqbc6avO7IatIC69M/view?usp=sharing

CHILD: https://lv-hyku-staging.notch8.cloud/concern/parent/ade4f58b-155a-4540-9a09-6e6eed5f1f49/images/79bded07-d012-4353-9f70-a1d72feda541

![Image](https://user-images.githubusercontent.com/1054448/203153077-0df835c3-877a-49fc-8bc1-c2fc66353f7b.png)

PARENT: (should have found 4 ocr results) https://lv-hyku-staging.notch8.cloud/concern/images/ade4f58b-155a-4540-9a09-6e6eed5f1f49?locale=en

![Image](https://user-images.githubusercontent.com/1054448/203153156-607057b2-138d-4138-b851-389ce1560d6d.png)

**Comment by orangewolf** [comment]
Related Merge Requests
---
[Draft: I149 parent child ocr search bug](https://gitlab.com/notch8/louisville-hyku/-/merge_requests/168)
!168
[i149 - fix timing issue that cause child ocr search bug](https://gitlab.com/notch8/louisville-hyku/-/merge_requests/170)
!170

**Comment by orangewolf** [comment]
Rachel September 2022
---
We uploaded a similar collection to our server - JPEGs with parent-level metadata and the expectation of OCR at the child level, with the same result.

**Comment by orangewolf** [comment]
Shana September 2022
---
Thanks for doing so. We’re on it!

**Comment by orangewolf** [comment]
Shana September 2022
---
@orangewolf Sample data for local dev: https://drive.google.com/file/d/1y-m6QhE_s6r_6LMVqbc6avO7IatIC69M/view?usp=sharing

**Comment by orangewolf** [comment]
Shana September 2022
---
Testing instructions: https://gitlab.com/notch8/louisville-hyku/-/merge_requests/170#expected-behaviortesting-instructions

Important note esp for testing on your dev server is linked above. 

**TLDR:**

Testing this w pre-existing works: 
1. May require reindexing works created manually. 
2. May require reimporting works created via bulkrax.

@rtkuehn https://github.com/rkuehn-uofl/hyku/pull/20

**Comment by orangewolf** [comment]
Diem September 2022
---
**This passes Notch8 QA:** See attached screen recording.

tested with:
- manually created parent work: https://lv-hyku-staging.notch8.cloud/concern/images/202209211004?locale=en
- attached child work that is a fileset with text: https://lv-hyku-staging.notch8.cloud/concern/parent/202209211004/images/202209211006

<details><summary>The Shank and Bone logo is the parent work while the cat scan is the attached child work. OCR search is working on child works when searching from both the parent and child UV.</summary>
![Screen_Recording_2022-09-21_at_10.10.46_AM](/uploads/f3a4001644747112aa4f7548e93032dc/Screen_Recording_2022-09-21_at_10.10.46_AM.mp4)
</details>

**Comment by orangewolf** [comment]
Randy September 2022
---
I've noticed that there a a number of failed to load resource map errors regarding the UV. It's difficult to replicate, but sometimes the shim.min.js.map error pops up a second time and stops the UV from functioning. What can we do to best resolve the missing map files?
This screenshot doesn't include the 2nd killer shim.map...


![Image](https://user-images.githubusercontent.com/1054448/203153991-b4b1fe0f-9922-481c-9dc4-14fea69a5c07.png)

**Comment by orangewolf** [comment]
Shana September 2022
---
Would you be able to share screens and show us in the meeting? @rtkuehnWould you be able to share screens and show us in the meeting? [@rtkuehn](https://github.com/rtkuehn)

**Comment by orangewolf** [comment]
Rachel September 2022
---
I'm in the process of deleting the yearbooks we had in the system since the changed code wasn't applied to them! But I can try to quickly upload that small sample of one yearbook

**Comment by orangewolf** [comment]
Shana September 2022
---
[small-yearbook.zip](/uploads/ca46ac4b6cc08cd61acb606f50f0f5f2/small-yearbook.zip) @rachel.howard

**Comment by orangewolf** [comment]
Rachel September 2022
---
We uploaded a PDF set (Law School WWII Newsletters) and the Macauley compound objects (JPEG sets) to our dev server, and search from UV worked for both!
